aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLCommandFactory.java

```diff
@@ -76,8 +88,7 @@ SqlCommand queryFor(@NonNull ModelSchema modelSchema,
      * @return the SQL command that encapsulates the UPDATE command
      */
     @NonNull
-    SqlCommand updateFor(@NonNull ModelSchema modelSchema,
-                         @NonNull QueryPredicate predicate) throws DataStoreException;
+    <T extends Model> SqlCommand updateFor(@NonNull ModelSchema modelSchema, @NonNull T item) throws DataStoreException;
```

Comment:
Looks like the predicate was not previously being unused? It wasn't in the JavaDoc either. Can you update the JavaDoc with the new param (and if any others were missed in this file)?

Reply:
Previously, the `QueryPredicate` was derived from a `Model` instance before being passed in.  This PR moves that logic into this method, so this class now derives the `QueryPredicate` from the `Model` instance.  

And yes, it does look like all the Javadoc in this interface is missing some params, so I will update them all.